src/daft-sql/src/planner.rs

```diff
@@ -66,30 +68,46 @@ impl Relation {
     }
 }
 
-#[derive(Clone)]
-pub struct SQLPlanner {
+struct GlobalPlanInfo {
```

Comment:
I think the naming 'global' is a bit misleading as it's really just the context of that current query, not a truly global context.

Can we rename this to `PlannerContext` or `SQLContext`